scripts/sanitize-mocha-results.js

```diff
@@ -18,7 +18,7 @@ function isAllowlistedEnv (key, value) {
     ['true', 'false', 'TRUE', 'FALSE', 'True', 'False'].includes(value)
     || value.length < 4
     // allow some envs that are not sensitive
-    || ['nodejs_version', 'CF_DOMAIN', 'SKIP_RELEASE_CHANGELOG_VALIDATION_FOR_BRANCHES', 'CIRCLE_PROJECT_REPONAME', 'CIRCLE_PR_REPONAME', 'CIRCLE_BRANCH', 'CIRCLE_PROJECT_USERNAME',
+    || ['nodejs_version', 'NODE_VERSION', 'CF_DOMAIN', 'SKIP_RELEASE_CHANGELOG_VALIDATION_FOR_BRANCHES', 'CIRCLE_PROJECT_REPONAME', 'CIRCLE_PR_REPONAME', 'CIRCLE_BRANCH', 'CIRCLE_PROJECT_USERNAME',
```

Comment:
I have no idea where this is all of a sudden coming from, but it's just the node version so seems harmless to let through